src/components/existing-request/check-status-gray-box.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-
-<v-row no-gutters class="bg-light-gray">
+  <v-row no-gutters class="bg-light-gray">
```

Comment:
Mostly indentation fix here (hide whitespace changes to view).

Also I cleaned up a bit of layout below.